### PR TITLE
Improving whole site by fixing some issues

### PR DESCRIPTION
**Pull request by ben221199**
This pull request fixes some problems, like:
 - Disappearing pages (see #77)
 - Disappearing projects using Multiformats
 - Removing outdated dependencies, eventually replacing them with alternatives.
 - Improving some title properties.
 - Improving sitemap.xml
 - Adding robots.txt and adding sitemap to it.

### CODE REVIEW

File: layouts/partials/header.html [hunk 1]
```diff
@@ -1,12 +1,10 @@
 {{ $isHomePage := eq .Title .Site.Title }}
-{{ $basePath := $.Scratch.Get "basePath" }}
-
 <header class="{{ if $isHomePage }}home-page{{ end }}">
   <div class="header-section">
-    <img alt="Multiformats Logo" id="logo" src="{{ $basePath}}logo.svg"/>
+    <img alt="Multiformats Logo" id="logo" src="{{ .Site.BaseURL }}logo.svg"/>
```
Comment (rvagg) [inline layouts/partials/header.html:4]: doesn't seem to be working, see logo missing @: https://bafybeihrib6fpgbrsaevura3prs2ckprmxdin3bscl7litbwevztosgf4u.on.fleek.co/multiaddr/

File: package.json [hunk 2]
```diff
@@ -1,18 +1,17 @@
 {
   "name": "multiformats-website",
   "version": "1.0.0",
-  "description": "multiformats website",
+  "description": "Multiformats website",
   "main": "list-multihashes.js",
   "dependencies": {
+    "clean-css": "^5.3.2",
+    "less": "^4.1.3",
```
Comment (rvagg) [inline package.json:8]: I know it doesn't really matter, but for consistency I think these should be moved to `devDependencies`

**Comment by rvagg** [comment]
this is really nice, thanks @ben221199; just a couple of items I've commented on inline

**Comment by ben221199** [comment]
Thanks for reviewing @rvagg. I will give a look at them.

File: package.json (hunk 2 above)
Comment (ben221199) [reply package.json]: This one is now fixed.

File: layouts/partials/header.html (hunk 1 above)
Comment (ben221199) [reply layouts/partials/header.html]: And this one is also fixed.

**Comment by ben221199** [comment]
I improved the code. The logo now works.

**Review by rvagg — APPROVED** [review]
wonderful, thanks for the work here; it's a shame about the base_url, but perhaps someone can come up with something more robust later

**Pull request merged**